bittensor/_axon/axon_impl.py

```diff
@@ -280,7 +281,8 @@ def finalize_codes_stats_and_logs():
                     inputs_x = deserialized_forward_tensors, 
                     synapses = synapses,
                     priority = priority,
-                    hotkey= request.hotkey
+                    hotkey = request.hotkey,
+                    timeout = self.forward_timeout - (clock.time() - start_time)
```

Comment:
One thing here; we will need a higher timeout for seq2seq requests. So we need to figure out which synapse type and set the corresponding timeout